.github/workflows/master-push-update-worker-base-image.yml

```diff
@@ -0,0 +1,40 @@
+name: Patavi | push | worker build & push
+
+on:
+  push:
+    branches: master
+
+jobs:
+  test:
+    runs-on: ${{ matrix.os }}
+
+    strategy:
+      matrix:
+        os: [ubuntu-20.04]
+
+    steps:
+      - uses: actions/checkout@v1
+      - uses: webfactory/ssh-agent@v0.4.1
+        with:
+          ssh-private-key: ${{ secrets.deployPrivateKey }}
+
+      - name: build worker uberjar
+        run: |
+          cd worker
+          lein uberjar
```

Comment:
kind of surprised that lein is just available out of the box

Reply:
Huh, I see https://github.com/actions/virtual-environments/blob/ubuntu20/20210517.1/images/linux/Ubuntu2004-README.md

very cool